src/view/merkle-drop.njk

```diff
@@ -11,61 +11,218 @@
+            <p>The tokens won’t be pushed onto accounts, but will instead have to be claimed from the Merkle drop smart contract.</p>
           </div>
-          <div class="column is-6">
-            Reward <strong>early contributors</strong>, e.g. validators, testers, developers and others.
+        </div>
+
+        <div class="columns">
+          <div class="column is-1"></div>
+          <div class="column is-10">
+            <h2>The Merkle drop will</h2>
           </div>
         </div>
 
-        <hr />
+        <div class="columns">
+          <div class="column is-1"></div>
+          <div class="column is-1 has-text-centered">
+            <img src="/img/icon-reward.svg">
+          </div>
+          <div class="column is-7">
+            Reward
+            <strong>early contributors</strong>, e.g. validators, testers, developers and others.
+            <hr/>
+          </div>
+        </div>
 
         <div class="columns">
-          <div class="column is-2 has-text-centered">
+          <div class="column is-1"></div>
+          <div class="column is-1 has-text-centered">
             <img src="/img/icon-target.svg">
           </div>
-          <div class="column is-6">
-            Target <strong>potential stakeholders</strong> of the Trustlines Network, e.g. individuals/ projects aligned with the mission of Trustlines, who may turn into users of the Trustlines technology stack.
+          <div class="column is-7">
+            Target
+            <strong>potential stakeholders</strong>
+            of the Trustlines Network, e.g. individuals/ projects aligned with the mission of Trustlines, who may turn into users of the Trustlines technology stack.
+            <hr/>
           </div>
         </div>
 
-        <hr />
-
         <div class="columns">
-          <div class="column is-2 has-text-centered">
+          <div class="column is-1"></div>
+          <div class="column is-1 has-text-centered">
             <img src="/img/icon-audience.svg">
           </div>
-          <div class="column is-6">
-            Make TLC available to a <strong>wide audience</strong> consisting of addresses and individuals from within the crypto and community currency ecosystems.
+          <div class="column is-7">
+            Make TLC available to a
+            <strong>wide audience</strong>
+            consisting of addresses and individuals from within the crypto and community currency ecosystems.
+            <hr/>
           </div>
         </div>
 
-        <p>&nbsp;</p>
+        <div class="columns">
+          <div class="column is-1"></div>
+          <div class="column is-10 mdt">
+            <h2>Merkle Drop Tutorial</h2>
+            <br/>
+            <center>
+              <iframe width="695" height="515" src="https://www.youtube-nocookie.com/embed/bHLYpZstZKs" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope;" allowfullscreen="allowfullscreen"></iframe>
```

Comment:
Nowadays it should be avoided to use `iframes` because of security and usability issues. Please try if something like the following is also satisfying.

```html
<object width="695" height="515" data="https://www.youtube-nocookie.com/embed/bHLYpZstZKs"></object>
```

Reply:
I could try this, but a clear problem there appears to be that it removes all the privacy preserving elements.

Reply:
Which?

Reply:
Actually the object worked well, thanks for that!